plugins/PrivacyManager/vue/src/OptOutCustomizer/OptOutCustomizer.vue

```diff
@@ -183,23 +302,25 @@ export default defineComponent({
         && this.fontColor !== ''
         && nearlyWhite(this.fontColor.slice(1));
     },
-    iframeUrl(): string {
+    codeBox(): string {
       if (this.piwikurl) {
-        const query = MatomoUrl.stringify({
-          module: 'CoreAdminHome',
-          action: 'optOut',
-          language: this.language,
-          backgroundColor: this.backgroundColor.slice(1),
-          fontColor: this.fontColor.slice(1),
-          fontSize: this.fontSizeWithUnit,
-          fontFamily: this.fontFamily,
-        });
-
-        return `${this.piwikurl}index.php?${query}`;
+        return this.code;
       }
-
       return '';
     },
+    iframeUrl(): string {
+      const query = MatomoUrl.stringify({
+        module: 'CoreAdminHome',
+        action: 'optOut',
+        language: this.language,
+        backgroundColor: this.backgroundColor.substr(1),
+        fontColor: this.fontColor.substr(1),
+        fontSize: this.fontSizeWithUnit,
+        fontFamily: this.fontFamily,
+        showIntro: (this.showIntro === true ? 1 : 0),
```

Comment:
Currently if `show intro` is not selected, the resulting url contains `&showIntro=`, might be better if it would contain `&showIntro=0` maybe, as otherwise it might look like the parameter could be simply removed, which isn't the case. Guess the values here needs to be strings to achieve that...